src/main/java/duke/ErrorHandler.java

```diff
@@ -0,0 +1,38 @@
+package duke;
+
+/**
+ * Provides static methods for handling various types of errors throughout the
+ * Duke application.
+ * This includes file not found errors, I/O errors, and number format errors.
+ */
+public class ErrorHandler {
```

Comment:
Using an error handler class is a very efficient way of handling the errors, nice!